spec/models/aspect_membership_spec.rb

```diff
@@ -7,25 +7,25 @@
 describe AspectMembership, :type => :model do
 
   describe '#before_destroy' do
-    before do
-      @aspect = alice.aspects.create(:name => "two")
-      @contact = alice.contact_for(bob.person)
+    let(:aspect)  { alice.aspects.create(name: "two") }
+    let(:contact) { alice.contact_for(bob.person) }
+    let(:am)      { alice.aspects.where(name: "generic").first.aspect_memberships.first }
```

Comment:
@jhass yeah, that's what we thought as well but we didn't want to change so much. But then, we'll do it :)